gapis/resolve/resource_data.go

```diff
@@ -74,7 +75,8 @@ func buildResources(ctx context.Context, p *path.Command) (*ResolvedResources, e
 	for k, v := range resources {
 		res, err := v.ResourceData(ctx, state)
 		if err != nil {
-			return nil, fmt.Errorf("Could not get data for resource %v", k)
+			log.E(ctx, "Could not get data for resource %v: %v", v.ResourceHandle(), err)
+			continue // Skip this resource, but sill return all the others.
```

Comment:
s/sill/still

With the above changed, we should actually pass the error down so it can be shown in the UI:

```go
	if err != nil {
		resourceData[k] = err
	} else {
		resourceData[k] = res
	}
```

and then at the end of `func (r *ResourceDataResolvable) Resolve(ctx context.Context)`:

```go
	id := r.Path.Id.ID()
	val, ok := res.resourceData[id]
	if !ok {
		return nil, fmt.Errorf("Cannot find resource with id: %v", id)
	}
	if err, isErr := val.(error); isErr {
		return nil, err
	} else {
 		return val, nil
	}
 }
```